kyaml/kio/kio.go

```diff
@@ -75,6 +75,11 @@ type Pipeline struct {
 
 	// Outputs are where the transformed Resource Configuration is written.
 	Outputs []Writer `yaml:"outputs,omitempty"`
+
+	// ContinueIfInputEmpty indicates should pipeline continue when the
+	// the input result is empty. This is useful when we want to run outputs
+	// even the input is empty.
+	ContinueIfInputEmpty bool
```

Comment:
If we decide to keep this variable, I prefer the names "ContinueOnEmptyInput" or simply "ContinueOnEmpty".

Reply:
Don't we prefer to avoid having to to think about this?
E.g.
```
~ $ cat /dev/null | sed 's/foo/bar/' | more; echo $?
0
~ $ 
```
works fine.  We don't have to do anything special with empty input. 
Yes, the sed process still starts, but it exits quickly.